code/game/objects/structures/watercloset.dm

```diff
@@ -290,6 +290,12 @@
 	anchored = TRUE
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 
+/obj/machinery/shower/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>You could <b>analyze</b> [src]'s temperature.</span>"
+	. += "<span class='notice'>The temperature valve seems to be held in place with <b>bolts</b>.</span>"
+	. += "<span class='notice'>It looks like you could <b>slice</b> [src] away from the wall.</span>"
```

Comment:
Some of this feels a little extra. I don't think the analyzer line needs to be included, it feels like more of a hidden interaction that you might stumble across. 
I think the temperature valve is more of something that you use the wrench to turn, it isn't really bolted into place. Could be better like
```
<span class='notice'>It looks like you could fit a wrench around the temperature valve.</span>
```

but otherwise, I think that's also fine as a sort of secret interaction. I'd leave that one up to you, but I really don't think we need the first one.

Reply:
Removing both then, it was really awkward to try to find something that fit for the temperature valve one.